packages/fluentui/docs/src/views/Accessibility.tsx

```diff
@@ -383,6 +383,37 @@ export default () => (
       `}
     />
 
+    <Header as="h3" content="Useful examples of labelling" />
+    <Header as="h4" content="Aria-labelledby refers to itself " />
+    <p>
+      In following example we want to get narrated from screen reader "delete Active members" button. Solution is
+      mentioned below, where `aria-labelledby` refers to the button `id` follows by heading element `id`. Screen reader
+      computes the accessible name in the order which is provided in `aria-labelledby`. The strategy was taken from the
+      {link(' Accessible Name and Description Computation 1.1', 'https://www.w3.org/TR/accname-1.1/#terminology')} page.
+    </p>
+    <CodeSnippet
+      value={`
+      <>
+      <Header as="h3" content="Active members" id="table-header" />
+      <Button
+        icon={<TableDeleteIcon />}
+        id="delete-button"
+        aria-label="delete"
+        aria-labelledby="delete-button table-header"
+      />
+    </>
+    `}
+    />
+
+    <Header as="h3" content="Accessibility name computation" />
+    <p>
+      Recommendation how screen reader is computing accessibility name is mentioned in
```

Comment:
The way how screen readers compute name and description is described in {link(' Accessible Name and Description Computation 1.1', 'https://www.w3.org/TR/accname-1.1/')} page. As the rules are quite complex, follow the page {' '}
      <Link to="button-name-computation"> Button name computation</Link> to see how these rules apply in concrete examples.

Reply:
applied changes